.pre-commit-config.yaml

```diff
@@ -1,34 +1,39 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: ''
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: 'v3.3.0'
     hooks:
-    -   id: end-of-file-fixer
+      - id: end-of-file-fixer
         exclude: 'setup.cfg'
-    -   id: trailing-whitespace
+      - id: trailing-whitespace
         exclude: 'setup.cfg'
-    -   id: debug-statements
--   repo: https://github.com/asottile/pyupgrade
-    rev: v2.7.3
+      - id: debug-statements
+  - repo: https://github.com/asottile/pyupgrade
+    rev: 'v2.7.3'
     hooks:
-    -   id: pyupgrade
+      - id: pyupgrade
         exclude: '(?:configobj/.*)'
         args:
-        - --py36-plus
--   repo: local
+          - --py36-plus
```

Comment:
Does `pyupgrade` support using  `pyproject.toml` for config options? My preference is to use  `pyproject.toml` as much as we can so when/if people run these tools manually, it uses the same config.

Reply:
@bdice sorry I was a bit late with this comment! I'll make an issue about moving args/config options out of the `.pre-commit-config.yaml` where possible

Reply:
@mikemhenry It is not possible to use `pyproject.toml`. More detail in my comment on #418. Good question!